csv-data-management/playbooks/playbooks.json

```diff
@@ -3,16 +3,16 @@
   "data": [
     {
       "@type": "WorkflowCollection",
-      "name": "CSV Data Management",
-      "description": null,
+      "name": "Sample - CSV Data Management - 1.0.0",
+      "description": "Sample playbooks for \"CSV Data Management\" connector. If you are planning to use any of the sample playbooks in your environment, ensure that you clone those playbooks and move them to a different collection, since the sample playbook collection gets deleted during connector upgrade and delete.",
```

Comment:
Recommendation: If you are planning to use/modify any of the sample playbook, clone respective playbook and move them to a different collection. Sample playbook collection gets deleted during connector upgrade and uninstall.

Reply:
This is added by framework not by playbook. No need to change